backend/src/api_parsers/models.py

```diff
@@ -3,35 +3,36 @@
 
 
 class Source(str, Enum):
-    SCOPUS = "SCOPUS"
     DBLP = "DBLP"
-    SCHOLAR = "SCHOLAR"
+    GoogleScholar = "Google Scholar"
+    Scopus = "Scopus"
 
 
 class Publication(BaseModel):
+    doi: str | None
     title: str
     year: int
     venue: str | None
-    citations: int | None
     abstract: str
-    source_api: Source
+    citation_count: int | None
     similarity_score: float | None
 
 
 class Author(BaseModel):
+    author_external_id: str
     first_name: str
     last_name: str
-    api_id: str
     affiliation: str | None
+    email: str | None
+    source: Source
     publication: Publication
 
     def get_attrs(self) -> dict:
         return {
-            'name': f'{self.first_name} {self.last_name}',
-            'affiliation': self.affiliation,
-            'title': self.publication.title,
-            'year': self.publication.year,
-            'source': self.publication.source_api,
-            'venue': self.publication.venue
+            "name": f"{self.first_name} {self.last_name}",
+            "affiliation": self.affiliation,
+            "title": self.publication.title,
+            "year": self.publication.year,
+            "source": self.source,
+            "venue": self.publication.venue,
```

Comment:
I would get rid of this function since it is redundant. You can call `model_dump()` method on a `pydantic.BaseModel` and it would to the same thing. The only difference is that the first and the last name would be assigned  to separate keys. But this is not a problem, we can easily handle it on the front-end.